src/store/src/InMemoryStore.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store;
+
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Store\Document\VectorDocument;
+
+final class InMemoryStore implements VectorStoreInterface
+{
+    /**
+     * @var VectorDocument[]
+     */
+    private array $documents;
+
+    public function add(VectorDocument ...$documents): void
+    {
+        foreach ($documents as $document) {
+            $this->documents[] = $document;
+        }
+    }
+
+    /**
+     * @param array{
+     *     maxItems?: positive-int
+     * } $options If maxItems is provided, only the top N results will be returned
+     */
+    public function query(Vector $vector, array $options = [], ?float $minScore = null): array
+    {
+        $currentEmbeddings = array_map(
+            callback: fn (VectorDocument $vectorDocument): array => [
+                'distance' => $this->cosineSimilarity($vectorDocument, $vector),
+                'document' => $vectorDocument,
+            ],
+            array: $this->documents,
```

Comment:
named arguments don't add value here or do I miss something?

```suggestion
            fn (VectorDocument $vectorDocument): array => [
                'distance' => $this->cosineSimilarity($vectorDocument, $vector),
                'document' => $vectorDocument,
            ],
            $this->documents,
```

Reply:
same for the others down below